synapse/storage/databases/main/roommember.py

```diff
@@ -670,6 +670,28 @@ async def get_users_who_share_room_with_user(
 
         return user_who_share_room
 
+    @cached(
+        max_entries=500000,
```

Comment:
The amount of entries here seems quite high, where did this number come from?

Reply:
I copy pasted this from the function above, I don’t know a good value for this, suggestions welcome.